### PR TITLE
Chart: Make PSS compliant

### PR DESCRIPTION
**Pull request by erkanerol**
Towards https://github.com/giantswarm/roadmap/issues/2970

Verified in CAPZ MC.

### CODE REVIEW

File: helm/dns-operator-azure/templates/psp.yaml [hunk 1]
```diff
@@ -33,4 +33,4 @@ spec:
   volumes:
     - "secret"
     - "configMap"
-{{- end }}
+{{- end }}
```
Comment (fiunchinho) [inline helm/dns-operator-azure/templates/psp.yaml:36]: Please add new line at the end of the file.

File: helm/dns-operator-azure/values.schema.json [hunk 2]
```diff
@@ -30,6 +30,19 @@
...
             "type": "string"
         },
+        "global": {
+            "type": "object",
+            "properties": {
+                "podSecurityStandards": {
+                    "type": "object",
+                    "properties": {
+                        "enforced": {
+                            "type": "boolean"
```
Comment (fiunchinho) [inline helm/dns-operator-azure/values.schema.json:40]: Would it better to add a default here? What do you think?

File: helm/dns-operator-azure/templates/psp.yaml (hunk 1 above)
Comment (erkanerol) [reply helm/dns-operator-azure/templates/psp.yaml]: fixed

File: helm/dns-operator-azure/values.schema.json (hunk 2 above)
Comment (erkanerol) [reply helm/dns-operator-azure/values.schema.json:41]: We haven't added it to other apps but it makes sense. Added here.

**Review by fiunchinho — APPROVED** [review]

**Pull request merged**